scripts/document.sh

```diff
@@ -15,13 +15,11 @@ fi
 
 
 OUTPUT=${OUTPUT:-documentation}
+DEFAULT_VERSION="x.x.x"
```

Comment:
Can the version default to something more accurate, like the `git describe` thing we used to use?